RplTaskTest.h

```diff
@@ -0,0 +1,9 @@
+class TestTask : public RplTask {
+	public:
+		int i = 0;
```

Comment:
If we are going to make changes, maybe a comment about what the purpose of this `i` value is :)